README.md
# Personal Website With Personal Projects


CHANGELOG.md
## Unreleased

### Enhancements

* Simplify year archive Liquid.
* Add documentation on how to downgrade theme.
* Improve greedy navigation's layout when JavaScript is disabled.
* Improve SEO include by grouping similar tags, reducing white-space, and adding `article:modified_time`. [#1456](https://github.com/mmistakes/minimal-mistakes/pull/1456)
* Minify `assets/js/lunr/lunr.js`.
* Improve calculation of Greedy navigation's `availableSpace`.
* Add Danish and Russian translations for new search strings. [#1472](https://github.com/mmistakes/minimal-mistakes/pull/1472) [#1477](https://github.com/mmistakes/minimal-mistakes/pull/1477)
* Indicate that archive title's are links with an underline.
* Remove `base_path` include from `/test` pages.
* Reduce font-size of page meta in list/grid items.
* Improve feature row styling when used with `archive` layout. [#1484](https://github.com/mmistakes/minimal-mistakes/issues/1484)

### Bug Fixes

* Allow`author` to accept an object or string. [#289](https://github.com/mmistakes/minimal-mistakes/issues/289)
* Fix syntax highlighting line number styling inconsistency. [#1467](https://github.com/mmistakes/minimal-mistakes/issues/1467)
* Fix author sidebar icon colors for dark skins. [#1482](https://github.com/mmistakes/minimal-mistakes/issues/1482)
* Remove misleading underline hover state on feature row items.
* Properly escape quotes in `site.social.name` and `site.name`. [#1485](https://github.com/mmistakes/minimal-mistakes/pull/1485)
* Fix typo in upgrading documentation. [#1487](https://github.com/mmistakes/minimal-mistakes/pull/1487)
* Replace `|` with HTML entity when used as title separator. [#760](https://github.com/mmistakes/minimal-mistakes/issues/760)

## [4.9.0](https://github.com/mmistakes/minimal-mistakes/releases/tag/4.9.0)

### Enhancements

* Add `show_overlay_excerpt` for disabling overlay image excerpt text. [#1436](https://github.com/mmistakes/minimal-mistakes/pull/1436)
* Update remote theme installation instructions in Quick Start Guide. [#1439](https://github.com/mmistakes/minimal-mistakes/pull/1439)
* Reduce visual weight of code blocks.
* Add Lunr.js Greek stemmer. [#1445](https://github.com/mmistakes/minimal-mistakes/pull/1445)
* Update Font Awesome 5 [SVG with JavaScript version](https://fontawesome.com/how-to-use/svg-with-js). [#1446](https://github.com/mmistakes/minimal-mistakes/pull/1446)
  * Note: if Font Awesome icons were used in the content of posts/pages or custom table of contents, find and replace any icons that have different names between version 4 and 5. Make sure to read the [complete list](https://fontawesome.com/how-to-use/upgrading-from-4#icon-name-changes-full) on Font Awesome's site.
* Reduce size of Lunr.js search JSON data and introduce `site.search_full_content` flag for limiting size of JSON file. [#1449](https://github.com/mmistakes/minimal-mistakes/pull/1449)
* Improve syntax highlighting styles. [#1450](https://github.com/mmistakes/minimal-mistakes/pull/1450)

### Bug Fixes

* Fix code block extra white-space when using [Jekyll's highlight tag](https://jekyllrb.com/docs/templates/#code-snippet-highlighting) with `linenos`. [#1437](https://github.com/mmistakes/minimal-mistakes/issues/1437)
* Round top-right corner of code block icon.
* Remove Lunr.js trimmer and bring back colons. [#1445](https://github.com/mmistakes/minimal-mistakes/pull/1445)
* Fix sticky `.sidebar` that overlaps main content when resizing viewport. [#1447](https://github.com/mmistakes/minimal-mistakes/issues/1447)

## [4.8.1](https://github.com/mmistakes/minimal-mistakes/releases/tag/4.8.1)

### Enhancements

* Add linkback functionality to author avatar and name in sidebar via `author.home`. [#1386](https://github.com/mmistakes/minimal-mistakes/pull/1386)
* Add Japanese localized UI text strings. [#1411](https://github.com/mmistakes/minimal-mistakes/pull/1411)
* Update Lunr.js to 2.1.5 [#1419](https://github.com/mmistakes/minimal-mistakes/pull/1419)

### Bug Fixes

* Fixed broken link to Staticman's page [#1422](https://github.com/mmistakes/minimal-mistakes/pull/1422)
* Fix Lunr search to work with number tags. [#1409](https://github.com/mmistakes/minimal-mistakes/issues/1409) [#1419](https://github.com/mmistakes/minimal-mistakes/pull/1419)

## [4.8.0](https://github.com/mmistakes/minimal-mistakes/releases/tag/4.8.0)

### Enhancements

* Open social share links in a new window. [#1357](https://github.com/mmistakes/minimal-mistakes/pull/1357)
* Remove Alexa.com verification due to retiring of "[Claim Your Site](https://support.alexa.com/hc/en-us/articles/219135887)" feature. [#1350](https://github.com/mmistakes/minimal-mistakes/issues/1350)
* Disable analytics in `development` environment. [#1362](https://github.com/mmistakes/minimal-mistakes/pull/1362)
* Disable comments in `development` environment. [#1363](https://github.com/mmistakes/minimal-mistakes/pull/1363)
* Exclude specific pages/posts from search index by adding `search: false` to the YAML Front Matter. [#1369](https://github.com/mmistakes/minimal-mistakes/pull/1369)
* Add optional `description` key to masthead links for clarifying their purpose with the `title` attribute. [#1380](https://github.com/mmistakes/minimal-mistakes/pull/1380)
* Incorporate site search into masthead. [#1383](https://github.com/mmistakes/minimal-mistakes/pull/1383)
* Update gem dependencies. [#1388](https://github.com/mmistakes/minimal-mistakes/pull/1388)

### Bug Fixes

* Fix `post.content` typo in `assets/js/lunr-en.js`. [#1354](https://github.com/mmistakes/minimal-mistakes/pull/1354)
* Fix "lunr-en.js:1 Uncaught SyntaxError: Unexpected token <" in `assets/js/lunr-en.js`. [#1356](https://github.com/mmistakes/minimal-mistakes/pull/1356)
* Rename Naver verification `naver_site_verification` to be consistent with other site variables.
* Fix button class in "Post with Table Of Contents" demo content. [#1368](https://github.com/mmistakes/minimal-mistakes/pull/1368)

projects/showcase/ido/gallery.php
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<title>I.DO-Gallery</title>
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="keywords" content="South African NGO, volunteering, matric rewrite, skills training, Registered NPO, Charity Work, Giving, Less privileged, Gap year, Volunteer work, fulfilling, Gauteng, Johannesburg, Trusted, Easy" />
<meta name="description" content="I.Do Foundation, specialist NGO making way for today's leaders. ">
<meta name="author" content="Mathemera">
<script type="application/x-javascript"> addEventListener("load", function() { setTimeout(hideURLbar, 0); }, false); function hideURLbar(){ window.scrollTo(0,1); } </script>
<!-- Custom Theme files -->
<link href="../css/bootstrap.css" type="text/css" rel="stylesheet" media="all">
<link href="../css/style.css" type="text/css" rel="stylesheet" media="all">   
<link rel="stylesheet" href="../css/flexslider.css" type="text/css" media="screen" />
<link rel="stylesheet" href="../css/jquery.flipster.css">
<!-- //Custom Theme files -->
<!-- font-awesome icons -->
<link href="../css/font-awesome.css" rel="stylesheet"> 
<!-- //font-awesome icons -->
<!-- js -->
<script src="../js/jquery-2.2.3.min.js"></script>  
<!-- //js -->
<!-- web-fonts -->  
<link href="//fonts.googleapis.com/css?family=Ubuntu" rel="stylesheet">
<link href="//fonts.googleapis.com/css?family=Yanone+Kaffeesatz:200,300,400,700" rel="stylesheet">
<link href='//fonts.googleapis.com/css?family=Roboto:400,100,100italic,300,300italic,400italic,500,500italic,700,700italic,900,900italic' rel='stylesheet' type='text/css'>
<!-- //web-fonts -->
<title>
  <?php 
    if (isset($_GET['album'])) {
	  echo $_GET['album'];
	} else {
	  echo 'Photo Gallery';
	}
  ?>
</title>

<!-- start gallery header --> 
<link rel="stylesheet" type="text/css" href="folio-gallery.css" />
<script type="text/javascript" src="js/jquery-1.9.1.min.js"></script>
<link rel="stylesheet" type="text/css" href="colorbox/colorbox.css" />
<!--<link rel="stylesheet" type="text/css" href="fancybox/fancybox.css" />-->
<script type="text/javascript" src="colorbox/jquery.colorbox-min.js"></script>
<!--<script type="text/javascript" src="fancybox/jquery.fancybox-1.3.1.min.js"></script>-->
<script type="text/javascript">
$(document).ready(function() {	
	
	// colorbox settings
	$(".albumpix").colorbox({rel:'albumpix'});
	
	// fancy box settings
	/*
	$("a.albumpix").fancybox({
		'autoScale	'		: true, 
		'hideOnOverlayClick': true,
		'hideOnContentClick': true
	});
	*/
});
</script>
<!-- end gallery header -->
</head>
<body>
	<!-- header -->
	<div class="headerw3l">
		<nav class="navbar navbar-default">
			<div class="container">
				<div class="navbar-header navbar-left">
					<h1><a href="/blog/index.php">I . DO <span><i>Youth.</i> <i class="logo-w3l2">Partnership.</i> <i class="logo-w3l3"> Growth.</i></span></a></h1>
				</div>
				<!-- navigation --> 
				<button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
					<span class="sr-only">Toggle navigation</span>
					<span class="icon-bar"></span>
					<span class="icon-bar"></span>
					<span class="icon-bar"></span>
				</button> 
				<!-- Collect the nav links, forms, and other content for toggling -->
				<div class="collapse navbar-collapse navbar-right" id="bs-example-navbar-collapse-1">					
					<ul class="nav navbar-nav navbar-left">
						<li><a href="/blog/index.php" class="link link--yaku"><span>H</span><span>O</span><span>M</span><span>E</span></a></li>
						<li><a href="about.html" class="link link--yaku"><span>A</span><span>B</span><span>O</span><span>U</span><span>T</span> <span>U</span><span>S</span></a></li>
						<li><a href="#" class="dropdown-toggle link link--yaku" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><span>S</span><span>E</span><span>R</span><span>V</span><span>I</span><span>C</span><span>E</span><span>S</span> <span class="caret"></span></a>
							<ul class="dropdown-menu">
								<li><a href="causes.html" class="link link--yaku"><span>C</span><span>A</span><span>U</span><span>S</span><span>E</span><span>S</span></a></li>
								<li><a href="Mentorship.html" class="link link--yaku"><span>M</span><span>E</span><span>N</span><span>T</span><span>O</span><span>R</span><span>S</span><span>H</span><span>I</span><span>P</span></a></li>
								<li><a href="education.html" class="link link--yaku"><span>E</span><span>D</span><span>U</span><span>C</span><span>A</span><span>T</span><span>I</span><span>O</span><span>N</span></a></li>
								<li><a href="downloads.html" class="link link--yaku"><span>D</span><span>O</span><span>W</span><span>N</span><span>L</span><span>O</span><span>A</span><span>D</span><span>S  </span></a></li>
								</ul>
						</li>
						<li class="active"><a href="gallery.html" class="link link--yaku"><span>G</span><span>A</span><span>L</span><span>L</span><span>E</span><span>R</span><span>Y</span></a></li>
						<li><a href="contact.php" class="link link--yaku"><span>C</span><span>O</span><span>N</span><span>T</span><span>A</span><span>C</span><span>T</span> <span>U</span><span>S</span></a></li>
						</ul>		
					<div class="clearfix"> </div>
				</div><!-- //navigation --> 
			</div>	
		</nav>		
	</div>	
	<!-- //header -->
<p>&nbsp;</p>

<div class="gallery">  
  <?php include "folio-gallery.php"; ?>
</div>   
	<!-- footer -->
	<div class="footer w3layouts">
		<div class="container">
			<div class="footer-agileinfo">
				 
				<div class="col-md-6 col-sm-3 footer-grid w3social">
					<h3>Quick links</h3>
					<ul>
						<li><a href="/blog/index.php">Home</a></li>
						<li><a href="about.html">About</a></li>
						<li><a href="volunteer.html">Become a Volunteer</a></li>
						<li><a href="donate.html">Donate</a></li>
						<li><a href="contact.php">Contact Us</a></li>
					</ul>
				</div>
				
				<div class="col-md-6 col-sm-3 footer-grid w3social">
					<h3>Social Media</h3>
					<ul>
						
						<li><a href="https://www.facebook.com/Ido-Foundation-South-Africa-457906351326592/"><i class="fa fa-facebook"></i>Facebook</a></li>
						<li><a href="https://twitter.com/idofoundationza"><i class="fa fa-twitter"></i>Twitter</a></li>
						
					</ul>
				</div>
				<div class="clearfix"> </div>
			</div> 
		</div>
	</div>
	
	<div class="copy-right"> 
		<div class="container">
		<p>NPO Number - 142004</p>
			<p>VAT Registration Number - 9373074195</p>
			<p>© 2018  I.DO Foundation . All rights reserved | Design by <a href=""> Fusse </a> powered by Folio Gallery</p>
		</div>
	</div  
	<!-- //footer --> 
	<!-- start-smooth-scrolling -->
	<script type="text/javascript" src="js/move-top.js"></script>
	<script type="text/javascript" src="js/easing.js"></script>	
	<script type="text/javascript">
			jQuery(document).ready(function($) {
				$(".scroll").click(function(event){		
					event.preventDefault();
			
			$('html,body').animate({scrollTop:$(this.hash).offset().top},1000);
				});
			});
	</script>
	<!-- //end-smooth-scrolling -->	
	<!-- smooth-scrolling-of-move-up -->
	<script type="text/javascript">
		$(document).ready(function() {
			/*
			var defaults = {
				containerID: 'toTop', // fading element id
				containerHoverID: 'toTopHover', // fading element hover id
				scrollSpeed: 1200,
				easingType: 'linear' 
			};
			*/
			
			$().UItoTop({ easingType: 'easeOutQuart' });
			
		});
	</script>
	<!-- //smooth-scrolling-of-move-up --> 
	<!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="js/bootstrap.js"></script>
</body>
</html>

projects/showcase/ido/folio-gallery.php
<?php 
// error_reporting (E_ALL ^ E_NOTICE);
// photo gallery settings
$mainFolder    = 'albums';   // folder where your albums are located - relative to root
$albumsPerPage = '12';       // number of albums per page
$itemsPerPage  = '12';       // number of images per page    
$thumb_width   = '150';      // width of thumbnails
//$thumb_height  = '85';       // height of thumbnails
$extensions    = array(".jpg",".png",".gif",".JPG",".PNG",".GIF"); // allowed extensions in photo gallery


// create thumbnails from images
function make_thumb($folder,$src,$dest,$thumb_width) {

	$source_image = imagecreatefromjpeg($folder.'/'.$src);
	$width = imagesx($source_image);
	$height = imagesy($source_image);
	
	$thumb_height = floor($height*($thumb_width/$width));
	
	$virtual_image = imagecreatetruecolor($thumb_width,$thumb_height);
	
	imagecopyresampled($virtual_image,$source_image,0,0,0,0,$thumb_width,$thumb_height,$width,$height);
	
	imagejpeg($virtual_image,$dest,100);
	
}

// display pagination
function print_pagination($numPages,$urlVars,$currentPage) {
        
   if ($numPages > 1) {
      
	   echo 'Page '. $currentPage .' of '. $numPages;
	   echo '&nbsp;&nbsp;&nbsp;';
   
       if ($currentPage > 1) {
	       $prevPage = $currentPage - 1;
	       echo '<a href="?'. $urlVars .'p='. $prevPage.'">&laquo;&laquo;</a> ';
	   }	   
	   
	   for( $e=0; $e < $numPages; $e++ ) {
           $p = $e + 1;
       
	       if ($p == $currentPage) {	    
		       $class = 'current-paginate';
	       } else {
	           $class = 'paginate';
	       } 
	       

		       echo '<a class="'. $class .'" href="?'. $urlVars .'p='. $p .'">'. $p .'</a>';
		  	  
	   }
	   
	   if ($currentPage != $numPages) {
           $nextPage = $currentPage + 1;	
		   echo ' <a href="?'. $urlVars .'p='. $nextPage.'">&raquo;&raquo;</a>';
	   }	  	 
   
   }

}


if (!isset($_GET['album'])) {

    // display list of albums
    $folders = scandir($mainFolder, 0);
	$ignore  = array('.', '..', 'thumbs');
		  
	$albums = array();
	$captions = array();
	$random_pics = array();
	  
    foreach($folders as $album) {
         
	    if(!in_array($album, $ignore)) {    
			 
		   array_push( $albums, $album );
			 
		   $caption = substr($album,0,20);
		   array_push( $captions, $caption );
			 
		   $rand_dirs = glob($mainFolder.'/'.$album.'/thumbs/*.*', GLOB_NOSORT);
           $rand_pic  = $rand_dirs[array_rand($rand_dirs)];
		   array_push( $random_pics, $rand_pic );
		  
		 }
		  
	 }

  
     if( count($albums) == 0 ) {
  
        echo 'There are currently no albums.';     
  
     } else {
  
		$numPages = ceil( count($albums) / $albumsPerPage );

        if(isset($_GET['p'])) {
      
	        $currentPage = $_GET['p'];
            if($currentPage > $numPages) {
               $currentPage = $numPages;
            }

         } else {
            $currentPage=1;
         } 
 
         $start = ( $currentPage * $albumsPerPage ) - $albumsPerPage;
	  
	     echo '<div class="titlebar">
                 <div class="float-left"><span class="title">Photo Gallery</span> - Albums</div>
			     <div class="float-right">'.count($albums).' albums</div>
              </div>';
	  
         echo '<div class="clear"></div>';
	  	  			 
	     for( $i=$start; $i<$start + $albumsPerPage; $i++ ) {
	  
	        if( isset($albums[$i]) ) {
			 		 			 
			    echo '<div class="thumb-album shadow">
				        
						<div class="thumb-wrapper">
						   <a href="'.$_SERVER['PHP_SELF'].'?album='. urlencode($albums[$i]) .'">
			                 <img src="'. $random_pics[$i] .'" width="'.$thumb_width.'" alt="" />
						   </a>	
					    </div>
						
						<div class="p5"></div>
					    
						<a href="'.$_SERVER['PHP_SELF'].'?album='. urlencode($albums[$i]) .'">
						<span class="caption">'. $captions[$i] .'</span>
						</a>
		            
					  </div>';
				  
		     }		  	  

	      }
	  
	      echo '<div class="clear"></div>';
  
          echo '<div align="center" class="paginate-wrapper">';
        	 
                 $urlVars = "";
                 print_pagination($numPages,$urlVars,$currentPage);
  
          echo '</div>';	   
   
     }
   

} else {

     // display photos in album
     $src_folder = $mainFolder.'/'.$_GET['album'];
     $src_files  = scandir($src_folder);

     $files = array();
     foreach($src_files as $file) {
        
		$ext = strrchr($file, '.');
        if(in_array($ext, $extensions)) {
          
		   array_push( $files, $file );
		  
		   
		   if (!is_dir($src_folder.'/thumbs')) {
              mkdir($src_folder.'/thumbs');
              chmod($src_folder.'/thumbs', 0777);
              //chown($src_folder.'/thumbs', 'apache'); 
           }
		   
		   $thumb = $src_folder.'/thumbs/'.$file;
           if (!file_exists($thumb)) {
              make_thumb($src_folder,$file,$thumb,$thumb_width); 
          
		   }
        
		 }
      
	  }
 

   if ( count($files) == 0 ) {

      echo 'There are no photos in this album!';
   
   } else {
   
      $numPages = ceil( count($files) / $itemsPerPage );

      if(isset($_GET['p'])) {
      
	     $currentPage = $_GET['p'];
         if($currentPage > $numPages) {
            $currentPage = $numPages;
         }

      } else {
         $currentPage=1;
      } 

   $start = ( $currentPage * $itemsPerPage ) - $itemsPerPage;

   echo '<div class="titlebar">
           <div class="float-left"><span class="title">'. $_GET['album'] .'</span> - <a href="'.$_SERVER['PHP_SELF'].'">View All Albums</a></div>
           <div class="float-right">'.count($files).' images</div>
         </div>';	  
   echo '<div class="clear"></div>';


   for( $i=$start; $i<$start + $itemsPerPage; $i++ ) {
		  
		  if( isset($files[$i]) && is_file( $src_folder .'/'. $files[$i] ) ) { 
	   
	        echo '<div class="thumb shadow">
	                <div class="thumb-wrapper">
					<a href="'. $src_folder .'/'. $files[$i] .'" class="albumpix" rel="albumpix">
				      <img src="'. $src_folder .'/thumbs/'. $files[$i] .'" width="'.$thumb_width.'" alt="" />
				    </a>
					</div>  
			      </div>'; 
      
	      } else {
		  
		    if( isset($files[$i]) ) {
			  echo $files[$i];
		    }
		   
		  }
     
    }
	   

     echo '<div class="clear"></div>';
  
     echo '<div align="center" class="paginate-wrapper">';
        	 
        $urlVars = "album=".urlencode($_GET['album'])."&amp;";
        print_pagination($numPages,$urlVars,$currentPage);
  
     echo '</div>';
	 
	 
   } // end else	 

}
?>

projects/showcase/ido/contact.php
<!--
Author: Fusse
Author URL:
-->
<!DOCTYPE html>
<html lang="en">
<head>
<title>I.DO - Contact Us</title>
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="keywords" content="South African NGO, volunteering, matric rewrite, skills training, Registered NPO, Charity Work, Giving, Less privileged, Gap year, Volunteer work, fulfilling, Gauteng, Johannesburg, Trusted, Easy" />
<meta name="description" content="I.Do Foundation, specialist NGO making way for today's leaders. ">
<meta name="author" content="Mathemera">
<script type="application/x-javascript"> addEventListener("load", function() { setTimeout(hideURLbar, 0); }, false); function hideURLbar(){ window.scrollTo(0,1); } </script>
<!-- Custom Theme files -->
<link href="css/bootstrap.css" type="text/css" rel="stylesheet" media="all">
<link href="css/style.css" type="text/css" rel="stylesheet" media="all">   
<link rel="stylesheet" href="css/flexslider.css" type="text/css" media="screen" />
<link rel="stylesheet" href="css/jquery.flipster.css">
<!-- //Custom Theme files -->
<!-- font-awesome icons -->
<link href="css/font-awesome.css" rel="stylesheet"> 
<!-- //font-awesome icons -->
<!-- js -->
<script src="js/jquery-2.2.3.min.js"></script>  
<!-- //js -->
<!-- web-fonts -->  
<link href="//fonts.googleapis.com/css?family=Ubuntu" rel="stylesheet">
<link href="//fonts.googleapis.com/css?family=Yanone+Kaffeesatz:200,300,400,700" rel="stylesheet">
<link href='//fonts.googleapis.com/css?family=Roboto:400,100,100italic,300,300italic,400italic,500,500italic,700,700italic,900,900italic' rel='stylesheet' type='text/css'>
<!-- //web-fonts -->
</head>
<body> 
	<!-- header -->
	<div class="headerw3l">
		<nav class="navbar navbar-default">
			<div class="container">
				<div class="navbar-header navbar-left">
					<h1><a href="/blog/index.php">I . DO <span><i>Youth.</i> <i class="logo-w3l2">Partnership.</i> <i class="logo-w3l3"> Growth.</i></span></a></h1>
				</div>
				<!-- navigation --> 
				<button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
					<span class="sr-only">Toggle navigation</span>
					<span class="icon-bar"></span>
					<span class="icon-bar"></span>
					<span class="icon-bar"></span>
				</button> 
				<!-- Collect the nav links, forms, and other content for toggling -->
				<div class="collapse navbar-collapse navbar-right" id="bs-example-navbar-collapse-1">					
					<ul class="nav navbar-nav navbar-left">
						<li><a href="/blog/index.php" class="link link--yaku"><span>H</span><span>O</span><span>M</span><span>E</span></a></li>
						<li><a href="about.html" class="link link--yaku"><span>A</span><span>B</span><span>O</span><span>U</span><span>T</span> <span>U</span><span>S</span></a></li>
						<li><a href="#" class="dropdown-toggle link link--yaku" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><span>S</span><span>E</span><span>R</span><span>V</span><span>I</span><span>C</span><span>E</span><span>S</span> <span class="caret"></span></a>
							<ul class="dropdown-menu">
								<li><a href="causes.html" class="link link--yaku"><span>C</span><span>A</span><span>U</span><span>S</span><span>E</span><span>S</span></a></li>
								<li><a href="Mentorship.html" class="link link--yaku"><span>M</span><span>E</span><span>N</span><span>T</span><span>O</span><span>R</span><span>S</span><span>H</span><span>I</span><span>P</span></a></li>
								<li><a href="education.html" class="link link--yaku"><span>E</span><span>D</span><span>U</span><span>C</span><span>A</span><span>T</span><span>I</span><span>O</span><span>N</span></a></li>
								<li><a href="downloads.html" class="link link--yaku"><span>D</span><span>O</span><span>W</span><span>N</span><span>L</span><span>O</span><span>A</span><span>D</span><span>S  </span></a></li>
								</ul>
						</li>
						<li><a href="gallery.html" class="link link--yaku"><span>G</span><span>A</span><span>L</span><span>L</span><span>E</span><span>R</span><span>Y</span></a></li>
						<li class="active"><a href="contact.php" class="link link--yaku"><span>C</span><span>O</span><span>N</span><span>T</span><span>A</span><span>C</span><span>T</span> <span>U</span><span>S</span></a></li>
						</ul>		
					<div class="clearfix"> </div>
				</div><!-- //navigation --> 
			</div>	
		</nav>		
	</div>	
	<!-- //header -->
	<!-- banner -->
	<div class="banner about-w3banner">  
		<div class="banner-img3">
			<div class="banner-w3text">
				<h2></h2>
			</div>
		</div> 	 
	</div>
	<!-- //banner -->
	<!-- contact -->
	<div class="contact w3-agileits">
		<div class="container">
			<h3 class="agileits-title1">Contact Us</h3>  
			<div class="contact-w3lsrow">
				<div class="col-md-6 contact-left wthree">
					<iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3023.9503398796587!2d-73.9940307!3d40.719109700000004!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x89c25a27e2f24131%3A0x64ffc98d24069f02!2sCANADA!5e0!3m2!1sen!2sin!4v1441710758555" allowfullscreen></iframe>
				</div>
				<div class="col-md-6 contact-right">
					<form action="#" method="post">
						<input type="text" name="name" placeholder="* Name" required="">
						<input type="text" name="emailad" placeholder="* Email" required="">
						<input type="text" name="phonenum" placeholder="* Phone number" required="">
						<textarea name="message" placeholder="* Message" required=""></textarea>
						<input type="submit" value="SEND">
					</form>
				</div>
				
				
<!-- Start PHP CODE FOR SENING EMAIL -->
		<?php
							if(isset($_POST['email'])) {
							 
								// EDIT THE 2 LINES BELOW AS REQUIRED
								$email_to = "info@idofoundation.co.za";
								$email_subject = "Email Query";
							 
								function died($error) {
									// your error code can go here
									echo "We are very sorry, but there were error(s) found within the form you submitted. ";
									echo "These errors appear below.<br /><br />";
									echo $error."<br /><br />";
									echo "Please fix these errors.<br /><br />";
									die();
								}
							 
							 
								// validation expected data exists
								if(!isset($_POST['name']) ||
									!isset($_POST['emailad']) ||
									!isset($_POST['phonenum']) ||
									!isset($_POST['message'])) {
									died('We are sorry, but there appears to be a problem with the form you submitted.');       
								}
							 
								 
							 
								$name = $_POST['name']; // required
								$emailad = $_POST['emailad']; // required
								$phonenum = $_POST['phonenum']; // not required
								$message = $_POST['message']; // required
							 
								$error_message = "";
								$email_exp = '/^[A-Za-z0-9._%-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,4}$/';
							 
							  if(!preg_match($email_exp,$emailad)) {
								$error_message .= 'The Email Address you entered does not appear to be valid.<br />';
							  }
							 
								$string_exp = "/^[A-Za-z .'-]+$/";
							 
							  if(!preg_match($string_exp,$name)) {
								$error_message .= 'The First Name you entered does not appear to be valid.<br />';
							  }
							 
							  
							  if(strlen($message) < 2) {
								$error_message .= 'Your message does not appear to be valid.<br />';
							  }
							 
							  if(strlen($error_message) > 0) {
								died($error_message);
							  }
							 
								$email_message = "Form details below.\n\n";
							 
								 
								function clean_string($string) {
								  $bad = array("content-type","bcc:","to:","cc:","href");
								  return str_replace($bad,"",$string);
								}
							 
								 
							 
								$email_message .= "Name: ".clean_string($name)."\n";
								$email_message .= "Email: ".clean_string($emailad)."\n";
								$email_message .= "Telephone: ".clean_string($phonenum)."\n";
								$email_message .= "Query: ".clean_string($message)."\n";
							 
							// create email headers
							$headers = 'From: '.$emailad."\r\n".
							'Reply-To: '.$emailad."\r\n" .
							'X-Mailer: PHP/' . phpversion();
							@mail($email_to, $email_subject, $email_message, $headers);  
							?>
							 
							<!-- include your own success html here -->
							 
							Thank you for contacting us. We will be in touch with you very soon.
							 
							<?php
							 
							}
							?>
		<!-- End PHP CODE FOR SENING EMAIL -->				
				
				<div class="clearfix"> </div>
			</div>
			<!-- address -->
			<div class="address-agileinfo"> 
				<ul>
					<li><i class="glyphicon glyphicon-map-marker" aria-hidden="true"></i>Mohlakeng Recreation Centre, Mohlakeng Randfontein, JHB 1759 , RSA</li>
					<li><i class="phon">Call Us :</i>  +27</li>
					<li><i class="glyphicon glyphicon-envelope" aria-hidden="true"></i><a href="mailto:info@idofoundation.co.za"> info@idofoundation.co.za</a></li>
				</ul>
			 </div>
			<!-- //address --> 
		</div>
	</div>
	<!-- //contact -->  
	<!-- footer -->
	<div class="footer w3layouts">
		<div class="container">
			<div class="footer-agileinfo">
				 
				<div class="col-md-6 col-sm-3 footer-grid w3social">
					<h3>Quick links</h3>
					<ul>
						<li><a href="/blog/index.php">Home</a></li>
						<li><a href="about.html">About</a></li>
						<li><a href="volunteer.html">Become a Volunteer</a></li>
						<li><a href="donate.html">Donate</a></li>
						<li><a href="contact.php">Contact Us</a></li>
					</ul>
				</div>
				
				<div class="col-md-6 col-sm-3 footer-grid w3social">
					<h3>Social Media</h3>
					<ul>
						
						<li><a href="https://www.facebook.com/Ido-Foundation-South-Africa-457906351326592/"><i class="fa fa-facebook"></i>Facebook</a></li>
						<li><a href="https://twitter.com/idofoundationza"><i class="fa fa-twitter"></i>Twitter</a></li>
						
					</ul>
				</div>
				<div class="clearfix"> </div>
			</div> 
		</div>
	</div>
	
	<div class="copy-right"> 
		<div class="container">
		<p>NPO Number - 142004</p>
			<p>VAT Registration Number - 9373074195</p>
			<p>© 2018  I.DO Foundation . All rights reserved | Design by <a href=""> Fusse.</a></p>
		</div>
	</div  
	<!-- //footer --> 
	<!-- start-smooth-scrolling -->
	<script type="text/javascript" src="js/move-top.js"></script>
	<script type="text/javascript" src="js/easing.js"></script>	
	<script type="text/javascript">
			jQuery(document).ready(function($) {
				$(".scroll").click(function(event){		
					event.preventDefault();
			
			$('html,body').animate({scrollTop:$(this.hash).offset().top},1000);
				});
			});
	</script>
	<!-- //end-smooth-scrolling -->	
	<!-- smooth-scrolling-of-move-up -->
	<script type="text/javascript">
		$(document).ready(function() {
			/*
			var defaults = {
				containerID: 'toTop', // fading element id
				containerHoverID: 'toTopHover', // fading element hover id
				scrollSpeed: 1200,
				easingType: 'linear' 
			};
			*/
			
			$().UItoTop({ easingType: 'easeOutQuart' });
			
		});
	</script>
	<!-- //smooth-scrolling-of-move-up --> 
	<!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="js/bootstrap.js"></script>
</body>
</html>